lib/ImportStatement.js

```diff
@@ -335,7 +335,7 @@ export default class ImportStatement {
     }
 
     const tabJoined = named.join(`,\n${tab}`);
-    const namedMultiLine = `{\n${tab}${tabJoined},\n}`;
+    const namedMultiLine = `{\n${tab}${tabJoined}\n}`;
```

Comment:
Most projects I work in prefer trailing commas. I often see this eslint rule turned on: https://eslint.org/docs/rules/comma-dangle

If we want to allow people to avoid dangling commas, we should add a new configuration option so that the default of adding a comma can be kept. Then, people can turn it off with config.